demo/assets/css/base.css

```diff
@@ -0,0 +1,104 @@
+body {
+    background: white;
+    font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+}
+
+/* TITLES */
+
+h3 {
+    font-size: 1rem;
+}
+
+/* TABLES */
+.table {
+    border: 1px solid;
+    width: 100%;
+}
+
+th {
```

Comment:
th, tr, td could be scoped inside the .table? .table th {}  & .table tr {} & .table tr td {}

Reply:
Can